server/src/test/java/org/elasticsearch/common/rounding/TimeZoneRoundingTests.java

```diff
@@ -514,6 +514,22 @@ public void testEdgeCasesTransition() {
         }
     }
 
+    public void testDST_Europe_Rome() {
+        // time zone "Europe/Rome", rounding to days. Rome had two midnights on the day the clocks went back in 1978, and
+        // timeZone.convertLocalToUTC() gives the later of the two because Rome is east of UTC, whereas we want the earlier.
```

Comment:
+1, I had to look up this change to be convinced, but I think this is the intuitively expected behaviour.